apps/prairielearn/src/pages/instructorStudentDetail/instructorStudentDetail.html.tsx

```diff
@@ -72,210 +67,41 @@ export function InstructorStudentDetail({
-            <button type="button" class="btn btn-sm btn-light" onClick={handleViewAsStudent}>
-              <i class="fas fa-user-graduate me-1" aria-hidden="true" />
-              View as student
-            </button>
-          )}
-        </div>
-        <div class="card-body">
-          <div class="d-flex align-items-center justify-content-between">
-            <h2 class="d-flex align-items-center">
-              {user?.name ?? enrollment.pending_uid}
-              {enrollment.status !== 'joined' && (
-                <EnrollmentStatusIcon type="badge" class="ms-2 fs-6" status={enrollment.status} />
-              )}
-            </h2>
-            {hasCourseInstancePermissionEdit && enrollmentManagementEnabled && (
-              <div class="d-flex gap-2 align-items-center">
-                {enrollment.status === 'joined' && (
-                  <form method="POST">
-                    <input type="hidden" name="__csrf_token" value={csrfToken} />
-                    <input type="hidden" name="__action" value="block_student" />
-                    <button type="submit" class="btn btn-sm btn-outline-danger">
-                      <i class="fas fa-user-slash me-1" aria-hidden="true" /> Block student
-                    </button>
-                  </form>
-                )}
-                {enrollment.status === 'blocked' && (
-                  <form method="POST">
-                    <input type="hidden" name="__csrf_token" value={csrfToken} />
-                    <input type="hidden" name="__action" value="unblock_student" />
-                    <button type="submit" class="btn btn-sm btn-outline-success">
-                      <i class="fas fa-user-check me-1" aria-hidden="true" /> Remove block
-                    </button>
-                  </form>
-                )}
-                {enrollment.status === 'invited' && (
-                  <form method="POST">
-                    <input type="hidden" name="__csrf_token" value={csrfToken} />
-                    <input type="hidden" name="__action" value="cancel_invitation" />
-                    <button type="submit" class="btn btn-sm btn-outline-secondary">
-                      <i class="fas fa-times me-1" aria-hidden="true" /> Cancel invitation
-                    </button>
-                  </form>
-                )}
-                {enrollment.status === 'rejected' && (
-                  <form method="POST">
-                    <input type="hidden" name="__csrf_token" value={csrfToken} />
-                    <input type="hidden" name="__action" value="invite_student" />
-                    <button type="submit" class="btn btn-sm btn-outline-primary">
-                      <i class="fas fa-user-plus me-1" aria-hidden="true" /> Re-invite student
-                    </button>
-                  </form>
-                )}
-              </div>
+    <TimezoneContext.Provider value={course_instance.display_timezone}>
+      <div class="container-fluid">
```

Comment:
Consider a non-fluid container? We don't remotely make good use of the whitespace on this page if it spans the full width. I didn't even see the "block"/etc. buttons the first few times I loaded the page because they were soooooo far away from the main content on the left.

Reply:
Fixed in b3e65a4d5375c1653d632e0b6e857e7801b9e71f